tools/version/main.go

```diff
@@ -0,0 +1,41 @@
+package main
+
+import (
+	"fmt"
+	"os/exec"
+	"regexp"
+)
+
+func main() {
+
+	// if the current branch is a release branch, parse the version number from the branch
+
+	cmd := exec.Command("git", "rev-parse", "--abbrev-ref", "HEAD")
+	output, err := cmd.CombinedOutput()
+	if err != nil {
+		fmt.Println("0.0.0")
```

Comment:
Do we want exit with an error? Have a different "error version" on every error flow? Anything else?

Reply:
I think I might go with "unknown"

Reply:
We don't think we can do that because semver will fail parsing a string that isn't in the "version format".